kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -0,0 +1,72 @@
+import { createTranslator } from 'kolibri.utils.i18n';
+
+export const coreStrings = createTranslator('CommonCoreStrings', {
+  // actions
+  cancelAction: 'Cancel',
```

Comment:
I don't know if this has been an issue, but there is always the possibility that the verb will need to agree with the elided direct object. e.g. "Delete [quiz]" may be translated differently from "Delete [user]" (bracket means it's left out/elided/implied.

Reply:
That's a great point. We haven't heard that feedback from from translators, but we honestly don't hear much from them. We have had the analogous case for adjectives, e.g. "Active [lesson]" vs "Active [quiz]".

I think the best strategy would be to name the actions with the elided concept, where appropriate. So for example, we would have independent strings for `deleteUserAction` and `deleteLessonAction`. This will have the effect of lowering the amount of re-use, but will ensure that when strings are reused they are more likely to be correct.

I don't think we'll be able to do this for strings where the elided word is not obvious. For example, `cancelAction`: cancel operation, cancel modal, cancel dialog, cancel activity...? There's no clearly correct object that end users will expect agreement with.

Reply:
I wonder how many places this is an issue in the codebase currently. 

If there are fewer than 3 cases of needing "Active [quiz]" on fewer than 3 separate components, then we would just define them directly on the component with a properly named action, like @indirectlylit mentioned ie, `quizActiveLabel`.